sw/device/tests/sim_dv/pwrmgr_deep_sleep_power_glitch_test.c

```diff
@@ -53,8 +53,10 @@ bool test_main(void) {
     wait_for_interrupt();
   } else {
     LOG_INFO("Checking reset status.");
-    rstmgr_testutils_post_reset(&rstmgr, kDifRstmgrResetInfoPowerUnstable, 0, 0,
-                                0, 0);
+    rstmgr_testutils_post_reset(
+        &rstmgr,
+        kDifRstmgrResetInfoPowerUnstable | kDifRstmgrResetInfoLowPowerExit, 0,
+        0, 0, 0);
     LOG_INFO("Reset status indicates a main power glitch reset");
```

Comment:
nit: should the log output be updated because the reset info bits changed?

Reply:
Well, the test was timing out so a failure in this check may not have been detected...